darwin/exporter/formats/nifti.py

```diff
@@ -195,24 +271,27 @@ def process_affine(affine):
     if isinstance(affine, str):
         affine = np.squeeze(np.array([ast.literal_eval(l) for l in affine.split("\n")]))
     elif isinstance(affine, list):
-        affine = np.array(affine).astype(np.float)
+        affine = np.array(affine).astype(float)
     else:
         return
     if isinstance(affine, np.ndarray):
         return affine
 
 
-def create_error_message_json(error_message, output_dir, image_id: str):
-    output_path = Path(output_dir) / f"{image_id}_error.json"
+def create_error_message_json(error_message, output_dir, image_id: str, slot_name: str):
+    output_path = Path(output_dir) / f"{image_id}_{slot_name}_error.json"
     if not output_path.parent.exists():
         output_path.parent.mkdir(parents=True)
     with open(output_path, "w") as f:
         native_json.dump({"error": error_message}, f)
 
+    return False
+
 
-def create_empty_nifti_file(volume_dims, affine, output_dir, image_id: str):
-    output_path = Path(output_dir) / f"{image_id}_empty.nii.gz"
+def create_empty_nifti_file(volume_dims, affine, output_dir, image_id: str, slot_name: str):
```

Comment:
Same typing comment

Reply:
Actually this 'create_empty_nifti_file' isn't being used anywhere so will remove this.